var/spack/repos/builtin/packages/powerapi/package.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Powerapi(AutotoolsPackage):
+    """This software is a reference implementation of the PowerAPI"""
+
+    homepage = "https://powerapi.sandia.gov/"
+    git      = "https://github.com/pwrapi/pwrapi-ref/tree/v1.1.1.git"
+
+    version('1.1.1', commit='93f66dfa29f014067823f2b790a1862e5841a11c', expand=False)
```

Comment:
This is generally used if you are fetching a single file with a URL. I don't think it's required here. Was there a reason you thought this was needed?

Reply:
What do you mean? Are you referring to `version('1.1.1', commit='93f66dfa29f014067823f2b790a1862e5841a11c', expand=False)` or the git url?